app/admin/widgets/calendar/assets/js/calendar.js

```diff
@@ -152,6 +153,7 @@
             var $this = $(this)
             var data = $this.data('ti.calendar')
             var options = $.extend({}, Calendar.DEFAULTS, $this.data(), typeof option == 'object' && option)
+            options.locale = options.locale.split('-').shift();
```

Comment:
I'd do this just before passing the options to `new FullCalendar.Calendar`

Reply:
Sure, done